Designs/TopLevel_oscillator.sch

```diff
@@ -29,10 +31,5 @@ hardcopy plot_1.svg v(Vout)
 hardcopy plot_2.svg v(Vin)
 shell display plot_1.svg &
 shell display plot_2.svg &
-plot Vin Vout
+wrdata osc_data Vout Vin
```

Comment:
are you sure you don't want to plot anymore here?

Reply:
yes, sure